cms/views.py

```diff
@@ -1,9 +1,31 @@
 from rest_framework import viewsets
 from rest_framework.decorators import action
 from rest_framework.response import Response
-from cms.models import Post
-from cms.serializers import PostSerializer
-from cms.permissions import UserPermission
+from cms.models import MenuItem, Post
+from cms.serializers import MenuItemShortSerializer, PostSerializer
+from cms.permissions import PostPermission
+
+
+class MenuItemViewSet(viewsets.ReadOnlyModelViewSet):
+    """
+    Obsluhuje API endpoint pre MenuItems
+    """
+    queryset = MenuItem.objects.order_by('-priority')
+    serializer_class = MenuItemShortSerializer
+
+    @action(methods=['get'], detail=False, url_path=r'on_site/(?P<site_id>\d+)')
```

Comment:
`on_site` by bolo vhodne nahradit za `on-site`, nepatri totiz medzi best practice pouzivat underscore v url

Reply:
Tu to viem nahradit ale čo tá druhá